purchase_request/tests/test_purchase_request_procurement.py

```diff
@@ -104,7 +104,6 @@ def test_procure_purchase_request(self):
         self.env["mail.activity"].search(
             [("activity_type_id", "=", activity.id)]
         ).unlink()
-        activity.unlink()
```

Comment:
The comments above these lines say 'An activity is created on the request even if the activity type was deleted'. If we don't delete the activity type anymore, we don't test the original requirement. Did this functionality break?

Reply:
That is great! That means that the activity type is guaranteed to exist. We can then drop the safeguarding from https://github.com/OCA/purchase-workflow/pull/2433/files#diff-556ada671b2bd79c2398ed16f2f6ad66bc3720e47b87d1ca4c232a367e72a122R44-R47. Can you remove or adjust the lines of this test that are about the situation in which this particular activity type is not found?